src/sentry/incidents/grouptype.py

```diff
@@ -79,7 +83,25 @@ class MetricAlertFire(GroupType):
     enable_escalation_detection = False
     detector_handler = MetricAlertDetectorHandler
     detector_validator = MetricAlertsDetectorValidator
-    detector_config_schema = {}  # TODO(colleen): update this
+    detector_config_schema = {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "description": "A representation of a metric alert firing",
```

Comment:
In general, `MetricAlertFire` is just the name the dan did real quick, and i expanded on cause lazy. how are you feeling about the name? should we change this to `MetricIssue` or something along those lines since it's an issue?

Reply:
There is currently already a [MetricIssuePOC](https://github.com/getsentry/sentry/blob/master/src/sentry/issues/grouptype.py#L615) group type from Snigdha's work, for now I think it's ok but later on we can consolidate the two.